MinecraftClient/config/ChatBots/VkMessager.cs

```diff
@@ -28,6 +28,27 @@
 /// - VK ChatId (typically 2000000001, etc.)
 /// - Bot's CommunityId
 /// </summary>
+public class VkMessager : ChatBot
+{
+    private VkLongPoolClient VkLongPoolClient { get; set; }
+    private readonly string ChatId;
+
+    public VkMessager(string vkToken, string chatId, string botCommunityId)
+    {
+        VkLongPoolClient = new VkLongPoolClient(vkToken, botCommunityId, ProcessMsgFromVk);
+        ChatId = chatId;
+    }
+
+    public override void Initialize()
+    {
+        LogToConsole("Bot enabled!");
+    }
+
+    private void ProcessMsgFromVk(string senderId, string peer_id, string text, string conversation_message_id, string id, string event_id)
```

Comment:
According to the description, the bot forward "messages between Minecraft and VKonrakte chats". Should this method do a `SendText()` call here to forward the message to Minecraft chat?

Also, should the VkMessager ChatBot override the `GetText()` method to capture messages and forward them to the VkLongPoolClient if they "start with dot (".")"?

Reply:
I think it should be left to users

Reply:
Ok, added a minimal working example just in case.